moon/config/moon-round1.json

```diff
@@ -39,7 +39,7 @@
       "rosmsg": {
           "driver": "rosmsg",
           "in": ["raw", "desired_speed", "tick", "stdout", "request_origin"],
-          "out": ["rot", "acc", "scan", "image", "pose2d", "sim_clock", "cmd", "origin"],
+          "out": ["rot", "acc", "scan", "image", "odo_pose", "sim_clock", "cmd", "origin"],
```

Comment:
why this rename? Note, that it does not take into account this `in/out`, only `register()` is what matters

Reply:
This is to introduce the idea that what is currently called 'pose2d' is only odometry's thought on what the pose may be. There are other sources of pose now, the idea is that there will be some kind of a fusion and only the best guess after all the sources are considered can be published under the generic 'pose' (or pose2d or pose3d) stream.